bugbug/models/testselect.py

```diff
@@ -833,9 +834,11 @@ def do_eval(
                     and result["caught_percentage_config_group"] is not None
                 )
 
-                message += f" In {percentage_caught_one_config_group}% of pushes we caught at least one config/group failure. On average, we caught {average_caught_percentage_config_group}% of all seen config/group failures."
-
-            print(message)
+            logger.info(
+                " In %d%% of pushes we caught at least one config/group failure. On average, we caught %d%% of all seen config/group failures.",
```

Comment:
```suggestion
                " In %d%% of pushes we caught at least one config/group failure. On average, we caught %s%% of all seen config/group failures.",
```
The average value could be float.